architecture/core_objects/kubernetes_model.adoc

```diff
@@ -220,8 +220,8 @@ documentation].
 
 == Service
 
-A service provides functionality to a set of pods running inside of a Kubernetes
-cluster, and are determined by the defined policy (sometimes called a
+A service provides functionality to a set of link:#pod[pods] running inside of a
+Kubernetes cluster, and are determined by the defined policy (sometimes called a
 micro-service) used to assess the set of pods. Pods can be added or taken away
```

Comment:
I'm fairly sure we don't want to confuse the meaning of "micro-service" here. I don't see anything being added by saying "(sometimes called a micro-service)".